src/routes/book.js

```diff
@@ -1,22 +1,30 @@
-module.exports = app => {
-  const books = require("../controllers/book.js");
+const { authJwt } = require("../middleware");
+const books = require("../controllers/book");
+const router = require("express").Router();
 
-  var router = require("express").Router();
+module.exports = app => {
+  app.use(function(req, res, next) {
+    res.header(
+      "Access-Control-Allow-Headers",
+      "x-access-token, Origin, Content-Type, Accept"
+    );
+    next();
+  });
 
   // Create a new Books
-  router.post("/", books.create);
+  router.post("/", [authJwt.verifyToken], books.create);
 
   // Retrieve all user Books
-  router.get("/", books.findAll);
+  router.get("/user/:id", [authJwt.verifyToken], books.findAll);
 
   // Retrieve a single Book with id
-  router.get("/:id", books.findOne);
+  router.get("/:id", [authJwt.verifyToken], books.findOne);
 
   // Update a Book with id
-  router.put("/:id", books.update);
+  router.put("/:id", [authJwt.verifyToken], books.update);
 
```

Comment:
Do you need the `[]` wraper?

Reply:
just if there are more than one middleware to process the request, in this case it is not necessary to use it